runbotics/runbotics-scheduler/src/scheduler-database/process/dto/create-process.dto.ts

```diff
@@ -13,6 +13,7 @@ export const createProcessSchema = z.object({
     definition: z.string(),
     isPublic: z.boolean(),
     isAttended: z.boolean().default(false),
+    executionInfo: z.string().trim().optional(),
```

Comment:
can this be an empty string as you put in dispatch with updateExecutionInfo: '' ? or maybe a null value?